src/scss/core/_type.scss

```diff
@@ -0,0 +1,26 @@
+h1, .h1 { line-height: $h1-line-height; }
+h2, .h2 { line-height: $h2-line-height; }
+h3, .h3 { line-height: $h3-line-height; }
+h4, .h4 {
+	line-height: $h4-line-height;
+	font-weight: $font-weight-normal;
+}
+
+@include media-breakpoint-up(lg) {
+	h1, .h1 {
+		@include font-size($h1-font-size-lg);
```

Comment:
why did you use the mixin for the font-size?

Reply:
I copied how the bootstrap does it. Currently the font-size mixin is just outputing the font size directly, unless we turn on the rfs (relative font size) function